packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -692,6 +694,18 @@ export default class TradeStore extends BaseStore {
         }
     };
 
+    // Get Turbos barrier Line Color Based on the Contract Type
+    getTurbosColor = type => {
+        switch (type) {
+            case 'TURBOSSHORT':
```

Comment:
@hamza-deriv 
is the type an upper case value?

Reply:
yes